tools/cleanup/cleanup.sh

```diff
@@ -73,39 +73,55 @@ delete_tvk_op() {
 
   # For Upstream OR in case if TVK installed on OCP using "helm"
   # Delete Triliovault-manager and Triliovault-operator using helm/label
-  if (helm list -A | grep -v REVISION | grep triliovault >/dev/null 2>&1); then
-    echo "Uninstalling Trilivault-manager"
-    tvm=$(helm list -A | grep -v REVISION | grep 'triliovault-[0-9]' | awk '{print $1}')
-    tvm_ns=$(helm list -A | grep -v REVISION | grep 'triliovault-[0-9]' | awk '{print $2}')
-    if [ -n "${tvm}" ]; then
-      helm uninstall "${tvm}" -n "${tvm_ns}"
+  # Fetch non-deuplicate namespace
+  tvm_ns=$(helm list -A | grep -v REVISION | grep 'triliovault' | awk '{print $2}' | uniq)
+  if [ -n "${tvm_ns}" ]; then
+    for ns in ${tvm_ns}; do
+
+      # Deleting Trilivault-manager CR
+      tvm_name=$(kubectl get triliovaultmanager --no-headers -n "${ns}" | awk '{print $1}')
+      echo "Patching triliovaultmanager CR ${tvm_name} in namespace ${ns}"
+      kubectl patch triliovaultmanager "${tvm_name}" -p '{"metadata":{"finalizers":[]}}' --type=merge -n "${ns}"
+      echo "Deleting triliovaultmanager CR ${tvm_name} in namespace ${ns}"
+      kubectl delete triliovaultmanager "${tvm_name}" -n "${ns}"
```

Comment:
for all `kubectl delete` commands add ` --force --grace-period=0 --timeout=5s` [if timeout exceeded command will return non-zero exit code, handle this]

first run `kubectl delete` then, run `kubectl patch`. Do this at all the places in this script.

Reply:
done